spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/WebOperationService.java

```diff
@@ -127,7 +128,11 @@ public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) {
                 return null;
             }
             if (operation.isBlocking()) {
-                ctx.blockingTaskExecutor().execute(() -> invoke(ctx, aggregatedReq, resFuture));
+                try {
+                    ctx.blockingTaskExecutor().execute(() -> invoke(ctx, aggregatedReq, resFuture));
```

Comment:
I think we can refactor this code a bit simpler.
```java
@Override
public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) {
    final Executor executor = operation.isBlocking() ? ctx.blockingTaskExecutor() : ctx.eventLoop();
    return HttpResponse.from(req.aggregate()
                                .thenApplyAsync(aggregatedReq -> invoke(ctx, aggregatedReq), executor));
}

private HttpResponse invoke(ServiceRequestContext ctx, AggregatedHttpRequest req) {
    final Map<String, Object> arguments = getArguments(ctx, req);
    final Object result = operation.invoke(new InvocationContext(SecurityContext.NONE, arguments));
    try {
        return handleResult(ctx, result, req.method());
    } catch (Throwable throwable) {
        return Exceptions.throwUnsafely(throwable);
    }
}
```

Reply:
Gentle ping @mauhiz. Any thought on this?

Reply:
On second thought, it seems overkill to refactor the code with this PR. Please ignore this review.